releases/2020-11/js.md

```diff
@@ -34,9 +38,65 @@ If you have a bug or feature request for one of the libraries, please post an is
 
 ## Release highlights
 
-### _Package name_
+---
+
+=== COPY THIS AND ADD THE INFORMATION OF YOUR PACKAGE: ===
+
+Keep in mind that:
+
+- Including the package name in the headers makes the URL links work for multiple packages.
+- The format of this file will be cleaned up once all of your proposals are in.
+
+---
+
+### _Project name_ 
+
+#### @azure/_package-name_@_version_ [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/<service-folder>/<package-folder>/CHANGELOG.md)
+
+(leave blank)
+
+##### Breaking Changes on @azure/_package-name_@_version_
+
+- _Add one or more, or remove the "Breaking Changes on ..." entire section._
+
+##### New Features on @azure/_package-name_@_version_
+
+- _Add one or more, or remove the "New Features on ..." section._
+
+##### Major Fixes on @azure/_package-name_@_version_
+
+- _Add one or more, or remove the "Major Fixes on ..." section._
+
+---
+
```

Comment:
```suggestion
### Service Bus

#### @azure/service-bus@7.0.0-preview.8 [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/CHANGELOG.md#700-preview8-2020-11-04)

##### Breaking Changes on @azure/service-bus@7.0.0-preview.8
- The `processError` passed to `Receiver.subscribe` now receives a `ProcessErrorArgs` instead of just an error. This parameter provides additional context that can make it simpler to distinguish
  errors that were thrown from your callback (via the `errorSource` member of `ProcessErrorArgs`) as well as giving you some information about the entity that generated the error.
- The methods to complete, abandon, defer and deadletter a message along with the method to renew message lock have been moved from the message to the receiver. With this, we now have additional validation to ensure that a peeked message cannot be used with these methods.
- Method and interface renames based on user studies and internal reviews:

  - The word "Message" is added to all methods and interfaces related to creating and sending a batch of messages for clarity based on user studies:
    - The `createBatch` method on the sender is renamed to `createMessageBatch`.
    - The interface `CreateBatchOptions` followed by the options that are passed to the `createBatch` method is renamed to `CreateMessageBatchOptions`.
    - The `tryAdd` method on the message batch object is renamed to `tryAddMessage`.
  - Renames to `ServiceBusMessage` and `CorrelationRuleFilter` fields to better align with the AMQP spec:
    - "properties" renamed to "applicationProperties".
  - "label" renamed to "subject".
  - The interface `CreateReceiverOptions` followed by options that are passed to `ServiceBusClient.createReceiver` method is renamed to `ServiceBusReceiverOptions`.
  - The interface `AcceptSessionOptions` followed by options that are passed to `ServiceBusClient` `acceptSession` and `acceptNextSession` methods is renamed to `ServiceBusSessionReceiverOptions`.
  - The property `maxAutoRenewLockDurationInMs` of interface `ServiceBusSessionReceiverOptions` is renamed to `maxAutoLockRenewalDurationInMs` to be consistent with a similar option for renewing messages.
  - The property `subQueue` in the options passed to the `createReceiver()` method is renamed to `subQueueType` to reflect that the value is restricted and not meant to contain any queue names. The corresponding type `SubQueue` is removed in favor of inlining the string literals that represent valid values.
- `SqlRuleFilter` interface "sqlExpression" changed from optional to required.
- `ServiceBusSender.scheduleMessages` method signature updated: `scheduledEnqueueTimeUtc` and `messages` parameters are swapped as the messages are the primary object that is being worked with.
- `NamespaceProperties` interface property "messageSku" type changed from "string" to string literal type "Basic" | "Premium" | "Standard" to reflect the limited types it supports.
- `NamespaceProperties` interface property "namespaceType" has been removed as it does not provide any value.
- Interfaces corresponding to the returned responses from the methods under the `ServiceBusAdministrationClient` such as `NamespacePropertiesResponse`, `QueueResponse`, `TopicRuntimePropertiesResponse` have been removed in favor of using generic type `WithResponse<T>` for a cleaner API surface.
- Updated the `update{Entity}` methods under `ServiceBusAdministrationClient` with relevant param names and types, more docs.
- The raw responses(`_response`) in the returned objects from any of the methods under the `ServiceBusAdministrationClient` have been updated to return only the `{request, status, headers}`, properties such as `parsedHeaders`, `parsedBody` have been removed.

#### New Features on @azure/service-bus@7.0.0-preview.8
- A helper method parseServiceBusConnectionString has been added which validates and parses a given connection string for Azure Service Bus. You can use this to extract the namespace and entityPath details from the connection string.
- Tracing, using [@azure/core-tracing](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-tracing/README.md), has been added for sending and receiving of messages.
- `viaPartitionKey` property of interface `ServiceMessageBus` has been removed until we implement the [Transactions feature of Service Bus](https://docs.microsoft.com/azure/service-bus-messaging/service-bus-transactions).
- Removed `AmqpAnnotatedMessage`, `AmqpMessageHeaders`, `AmqpMessageProperties` interfaces in favour of the ones from `@azure/core-amqp`. This is part of the move from `@azure/core-amqp` version update from 1.1.x to 2.0.0-beta.y. As part of this, `userId` will not be made available as part of `AmqpMessageProperties` until its type is fixed in the upstream `rhea` library.

#### Major fixes on @azure/service-bus@7.0.0-preview.8
- Internal improvement - For the operations depending on $management link such as peek or lock renewals, the listeners for the "sender_error" and "receiver_error" events were added to the link for each new request made before the link is initialized which would have resulted in too many listeners and a warning such as `MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 sender_error listeners added to [Sender]. Use emittr.setMaxListeners() to increase limit(same for receiver_error)`. This has been improved such that the listeners are reused.
- 
```

Reply:
Under Breaking Changes, I did not mention all the method/interface renames or updates since there are too many, pointed to the actual changelog instead for those changes.

Reply:
Updated based on this! I'll resolve.